app/controllers/administrator/users_controller.rb

```diff
@@ -2,10 +2,12 @@
 
 module Administrator
   class UsersController < ApplicationController
-    before_action :set_user, only: %i[edit update]
+    before_action :set_user, only: %i[edit update resend_invite]
 
     def index
       @users = current_local_authority.users
+      @confirmed_users = current_local_authority.users.select { |u| u.confirmed? }
+      @unconfirmed_users = current_local_authority.users.select { |u| u.unconfirmed? }
```

Comment:
I think we could add some scopes to the user model

`scope :confirmed, -> { where.not(confirmed_at: nil) }`
`scope :unconfirmed, -> { where(confirmed_at: nil) }`

Then use the scoped association rather than a select query